tools/clang-tidy.yml

```diff
@@ -14,5 +14,17 @@ services:
 
     command: sh -c "run-clang-tidy.sh"
 
+  clang-tidy-action:
```

Comment:
Why did you insert `clang-tidy` a second time. What is the difference between `clang-tidy` and `clang-tidy-action` and why are both required?

Reply:
clang-tidy is used by the user (tools/run-clang-tidy), if they want to check locally. clang-tidy-action is run by the GitHub action, because it needs access to the output in the cmake dir and hence binds that to /tmp/build. This directory is not required to exist if a user runs clang-tidy.